src/command.ts

```diff
@@ -136,7 +136,7 @@ export default abstract class Command {
 
   static set globalFlags(flags: Interfaces.FlagInput<any>) {
     this._globalFlags = Object.assign({}, this.globalFlags, flags)
-    this.flags = this.globalFlags
+    this.flags = {} // force the flags setter to run
```

Comment:
@nsamsami Not sure I understand the "force the setter to run" as before and after  changes  are both simple assignments?

Reply:
In both cases, the setter was being executed. But I changed it to an empty object since the flags setter is already adding `this.globalFlags`. In other words, it's duplicative to pass in `this.globalFlags`

I added the comment in case anyone looked at this and thought we were just setting `this.flags` to an empty object

Reply:
🤔 
Yeah, this comment seems kinda weird. Could you elaborate @mdonnalley?